src/App.jsx

```diff
@@ -1,107 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentById(id) {
+  return commentsFromServer.filter(comment => comment.postId === id) || null;
```

Comment:
Array.filter always return an array
```suggestion
  return commentsFromServer.filter(comment => comment.postId === id);
```